build-configs/aschaffenburg/theme/fonts.js

```diff
@@ -4,9 +4,9 @@ import type { FontsType } from '../../common/theme/fonts'
 import { commonFonts } from '../../common/theme/fonts'
 
 const fonts: FontsType = {
-  decorativeFontBold: 'Raleway-Bold',
-  decorativeFontRegular: 'Raleway-Regular',
-  decorativeFontFamily: '\'Raleway\', \'Lateef\', sans-serif',
+  decorativeFontBold: 'VarelaRound-Regular',
+  decorativeFontRegular: 'VarelaRound-Regular',
+  decorativeFontFamily: '\'Varela Round\', \'Raleway\', \'Lateef\', sans-serif',
```

Comment:
```suggestion
  decorativeFontFamily: '\'Varela Round\', \'Lateef\', sans-serif',
```

Having Raleway here is equal to using the system default. So I'd omit it here.